pages/_type/_id/version.vue

```diff
@@ -142,7 +142,7 @@
       <div v-else-if="isEditing" class="input-group">
         <button
           class="iconified-button brand-button"
-          :disabled="!$nuxt.$loading"
+          :disabled="$nuxt.$loading && !$nuxt.$loading.show"
```

Comment:
This seems to have made versions uneditable, I suggest testing the other similar changes to ensure they do what you intended